lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -34,7 +35,7 @@ class UnusedBytes extends Audit {
    * @return {string}
    */
   static bytesToKbString(bytes) {
-    return Math.round(bytes / KB_IN_BYTES).toLocaleString() + ' KB';
+    return Util.formatBytesToKB(bytes);
```

Comment:
Hm, are you sure? With `0` decimal places in some cases it will lead to misleading results e.g. instead of having _0.5 KB (50%)_ it will show _0 KB (50%)_.